LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -2168,8 +2168,9 @@ strCharacterName="Resistance MEC"
 [Reward_Rebel X2RewardTemplate]
 DisplayName="Resistance Personnel"
 
-[Reward_ResistanceMEC X2RewardTemplate]
+[Reward_ResistanceMec X2RewardTemplate]
```

Comment:
I think this is case insensitive, but I don't understand why you want to change this when the reward name is explicitly 'Reward_ResistanceMEC'.

Reply:
That was just leftover from me making the reward template when I wasn't even aware that it existed in the first place.